### PR TITLE
cloud: ovirt: add kernel boot parameters to vms module

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

ovirt_vms

##### ANSIBLE VERSION

```
ansible 2.3.0
```

##### SUMMARY

This patch adds three new parameters to `ovirt_vms` modules, and it's following:

 - kernel_path
 - initrd_path
 - kernel_params

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina @ryansb

**Comment by machacekondra** [comment]
shipit

File: lib/ansible/modules/cloud/ovirt/ovirt_vms.py [hunk 1]
```diff
@@ -232,6 +232,20 @@
             - "C(nic_name) - Set name to network interface of Virtual Machine."
             - "C(nic_on_boot) - If I(True) network interface will be set to start on boot."
         version_added: "2.3"
+    kernel_path:
+        description:
+            - "Path to kernel image to boot the virtual machine."
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_vms.py]: I'd rephrase to: "Path to a kernel image used to boot the virtual machine"

File: lib/ansible/modules/cloud/ovirt/ovirt_vms.py [hunk 2]
```diff
@@ -232,6 +232,20 @@
...
+            - "Kernel image must be stored on either the ISO domain or on the host's storage."
+        version_added: "2.3"
+    initrd_path:
+        description:
+            - "Path to a ramdisk to be used with the kernel."
+            - "Ramdisk image must be stored on either the ISO domain or on the host's storage."
+        version_added: "2.3"
+    kernel_params:
+        description:
+            - "Kernel command line parameter strings to be used with the defined kernel on boot."
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_vms.py]: I'd rephrase: "Kernel command line parameters (formatted as string) to be used with the kernel specified by `kernel_path` option."

File: lib/ansible/modules/cloud/ovirt/ovirt_vms.py [hunk 3]
```diff
@@ -232,6 +232,20 @@
...
             - "C(nic_on_boot) - If I(True) network interface will be set to start on boot."
         version_added: "2.3"
+    kernel_path:
+        description:
+            - "Path to kernel image to boot the virtual machine."
+            - "Kernel image must be stored on either the ISO domain or on the host's storage."
+        version_added: "2.3"
+    initrd_path:
+        description:
+            - "Path to a ramdisk to be used with the kernel."
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_vms.py]: I'd rephrase: "Path to an initial ramdisk to be used with the kernel specified by `kernel_path` option".

**Review by mwperina — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/ovirt/ovirt_vms.py (hunk 1 above)
Comment (machacekondra) [reply lib/ansible/modules/cloud/ovirt/ovirt_vms.py]: done

File: lib/ansible/modules/cloud/ovirt/ovirt_vms.py (hunk 3 above)
Comment (machacekondra) [reply lib/ansible/modules/cloud/ovirt/ovirt_vms.py]: done

File: lib/ansible/modules/cloud/ovirt/ovirt_vms.py (hunk 2 above)
Comment (machacekondra) [reply lib/ansible/modules/cloud/ovirt/ovirt_vms.py]: done

**Comment by abadger** [comment]
Merged to devel.  thanks @machacekondra and @mwperina !
